doc/nrf/security/tfm/processing_environments.rst

```diff
@@ -53,7 +53,7 @@ You can use Trusted Firmware-M to increase the security level of the application
 
 You can use the PSA Secure Storage API exposed by TF-M to store secrets in memory not accessible to the user application.
 In addition, the I2C peripheral can be designated as secure to prevent access from the non-secure application and only expose a defined interface towards the user application.
-For example, you can define this interface to allow the user application to perform a specific task (such as retrieving the value of a sensor, locking or unlocking a door, and so on) only under certain conditions that are decided by the code running within the associated partition in TF-M.
+For example, you can define this interface to allow the user application to perform a specific task, such as retrieving the value of a sensor, locking or unlocking a door, or triggering an alarm, only under certain conditions determined by the code running within the associated partition in TF-M.
```

Comment:
I don't like this edit.
By the way, how did this pass the Vale check for 30 words per sentence? :D
```suggestion
For example, you can define this interface to allow the user application to perform a specific task only under certain conditions determined by the code running within the associated partition in TF-M.
Such tasks can include retrieving the value of a sensor, locking or unlocking a door, triggering an alarm, among others. 
```